.github/workflows/review-summary.yml

```
@@ -0,0 +1,142 @@
+      github.event.review.state == 'changes_requested' ||
+      github.event.review.state == 'commented'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Get Review Details
+        id: review
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.10)

"github.event.review.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)